frontend/shared/src/error-handler/error-handler.ts

```diff
@@ -6,16 +6,21 @@ import { isError } from 'shared/utils/type-guards';
 type Params = {
   error: Error | unknown;
   t: TFunction;
+  pathname: string;
   goToPage: GoToPageFunction;
   onAuthError?: () => void;
   onServerError?: () => void;
   onCommonError?: () => void;
 };
 
 const handleError = (args: Params): void => {
-  const { error, t, goToPage } = args;
+  const { error, t, pathname, goToPage } = args;
   const {
-    onAuthError = () => goToPage('/login?sessionExpired=true'),
+    onAuthError = () => {
+      if (pathname !== '/login') {
+        goToPage('/login?sessionExpired=true');
```

Comment:
Now it goes to the `/login?sessionExpired=true` page when opening the employer UI by going to e.g. https://localhost:3000, right? I mean wouldn't it work the same in production also if employers would open the https://kesaseteli.hel.fi/ page and they weren't logged in yet they would get the "Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään" error notification rather than the "Palvelun käyttäminen edellyttää vahvaa tunnistautumista" info notification? Just thinking whether this is ok.

Reply:
I think that the backend will return 401 on both cases, so it's impossible to make the difference on ui-side. So it will show the `Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään` error on both cases, when the session exists but expired, or session didnt exist in the first place.

Reply:
Alright, this is another problem so this doesn't have to be tackled in this PR, IMHO we can leave it be like this now.